lightning_pose_app/ui/extract_frames.py

```diff
@@ -558,9 +593,10 @@ def _render_streamlit_fn(state: AppState):
             st.warning("waiting for existing extraction to finish")
 
         if state.st_submits > 0 and not st_submit_button and not state.run_script_video_random:
-            proceed_str = "Please proceed to the next tab to label frames."
-            proceed_fmt = "<p style='font-family:sans-serif; color:Green;'>%s</p>"
-            st.markdown(proceed_fmt % proceed_str, unsafe_allow_html=True)
+            with st.empty():
```

Comment:
does this work?

Reply:
Amended. cant resolve so please delete comment after reviewing